tools/ci_build/github/azure-pipelines/orttraining-py-packaging-pipeline-cpu.yml

```diff
@@ -0,0 +1,76 @@
+trigger: none
+
+stages:
+- stage: Python_Packaging
+
+  jobs:
+    - job: Linux_Training_CPU_Wheels
+      timeoutInMinutes: 60
+      workspace:
+        clean: all
+      pool: Onnxruntime-Linux-GPU-NV6
+
+      variables:
+        PythonVersion: '3.8'
+        DockerFile: 'Dockerfile.manylinux2014_training_cuda10_2'
```

Comment:
'Dockerfile.manylinux2014_training_cuda10_2' for CPU too?

Reply:
the cpu docker cannot be used for training build. Use training docker instead of creating a new docker.

Reply:
changed to cpu docker